ios-whitelist.json

```
@@ -352,6 +352,10 @@
     {
       "name": "Bugsnag",
       "version": "^~>\\s?5.[0-2][0-9]+"
+    },
+    {
+      "name": "MLSecurity",
+      "version": "^~>\\s?[0-1].[0-9]+"
```

@brunofamigliettimeli deberías dejar fijo el major:
```suggestion
      "version": "^~>\\s?1.[0-9]+"
```